src/main/java/com/google/devtools/build/lib/remote/GrpcRemoteExecutor.java

```diff
@@ -252,4 +252,8 @@ public void close() {
     }
     channel.release();
   }
+
+  RemoteRetrier getRetrier() {
+    return this.retrier;
```

Comment:
is this needed at all? I don't see this gets used anywhere or in the initerface

Reply:
it is used test methods inside RemoteModuleTest class.